src/platform/packages/shared/kbn-scout/src/config/discovery/search_configs.ts

```diff
@@ -88,3 +92,56 @@ export const getScoutPlaywrightConfigs = (searchPaths: string[], log: ToolingLog
 
   return pluginsWithConfigs;
 };
+
+export const validateWithScoutCiConfig = (
```

Comment:
`validateWithScoutCiConfig` has rather "involved" (error prone) logic. Should we write unit tests for it?

Reply:
added in d18693aa67d47970dd1859a811df7c45621c2210

Reply:
The tests are looking great. Thank you!